rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -71,61 +75,56 @@ def _get_packages_for_abipkgdiff(cls, input_structure=None):
         return debug_package, rest_packages
 
     @classmethod
+    def _find_debuginfo(cls, debug, pkg):
+        name = split_nevra(os.path.basename(pkg)).name
+        debuginfo = '{}-debuginfo'.format(name)
+        find = [x for x in debug if split_nevra(os.path.basename(x)).name == debuginfo]
+        if find:
+            return find[0]
+        srpm = RpmHelper.get_info_from_rpm(pkg, rpm.RPMTAG_SOURCERPM)
+        debuginfo = '{}-debuginfo'.format(split_nevra(srpm).name)
+        find = [x for x in debug if split_nevra(os.path.basename(x)).name == debuginfo]
+        if find:
+            return find[0]
+        return None
+
+    @classmethod
     def run_check(cls, result_dir):
         """Compares old and new RPMs using pkgdiff"""
```

Comment:
Not really related to this commit, but the comment could be fixed to the correct `abipkgdiff` name.